app/Http/Controllers/Auth/LoginController.php

```diff
@@ -49,7 +49,7 @@ protected function authenticated(Request $request, $user)
             return redirect()->route('apply.create');
         }
 
-        return redirect('/flights');
+        return redirect()->intended('/flights');
```

Comment:
Spicy, what does this do? Looks cool.

Reply:
Say the user has clicked on a link navigating them to `/flights/4` and they have to login, it will try to redirect them back to `/flights/4` if not it will put them to `/flights`